_posts/2020-09-09-#18267.md

```diff
@@ -10,14 +10,46 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
+This pull request implements the portion of
```

Comment:
Can you make this change to the PR metadata? The script doesn't yet parse "BIP" without changing it to "Bip".
```diff
-title: "Bip-325: Signet [consensus]"
+title: "BIP-325: Signet [consensus]"
```

Reply:
done